index.md

```diff
@@ -9,13 +9,13 @@ Checkout buttons are configurable (see options below), and multiple checkout but
 To include a checkout button on a page, you will need a checkout form-id. Once you have a form-id, paste this javascript into your site where you would like the form to appear:
 
 ```markdown
-<script id="ps-script" src="https://checkout.paymentspring.com/js/paymentspring.js" formid="cb7eae32763131ade35f"></script>
+<script src="https://checkout.paymentspring.com/js/paymentspring.js" formid="cb7eae32763131ade35f"></script>
 ```
 Where `cb7eae32763131ade35f` is your form-id.
 
 ### Options
 
-static amount:
+static amount (with ach):
```

Comment:
The 'static distribution' example below is also ach.

Reply:
updated!